src/docfx/lib/HtmlUtility.cs

```diff
@@ -319,7 +319,24 @@ public static string CreateHtmlMetaTags(JObject metadata)
                 }
                 else
                 {
-                    content = value.Type == JTokenType.Boolean ? (bool)value ? "true" : "false" : value.ToString();
+                    switch (value.Type)
+                    {
+                        case JTokenType.Boolean:
+                        {
+                            content = (bool)value ? "true" : "false";
+                            break;
+                        }
+                        case JTokenType.Date:
+                        {
+                            content = JsonUtility.Serialize(value).Trim('"');
```

Comment:
This would still not preserve the original string, it just reformats the date that happened to match loc inputs. The fix should disable JSON.NET automatic Date parsing in the first place.

Reply:
Updated